calibrators/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,121 @@
+| `apriltag_max_convergence_translation`        | `double`              | `2.0`                            | Maximum Threshold in pixels for corners Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                                                      |
+| `apriltag_new_hypothesis_translation`         | `double`              | `20.0`                           | Translation threshold in pixels for generating a new hypothesis in apriltag tracking.                                                                                                                                                                                   |
+| `apriltag_measurement_noise_translation`      | `double`              | `0.2`                            | The square of this value in meters is part of the input for the measurement noise covariance matrix (R) in the Kalman filter.                                                                                                                                           |
+| `apriltag_process_noise_translation`          | `double`              | `0.02`                           | The square of this value in meters is part of the input for the process noise covariance matrix (Q) in the Kalman filter.                                                                                                                                               |
+
+## Requirements
+
+### lidartag
+
+To perform camera-lidar calibration using this tool, it is necessary to prepare lidartags and lidars with intensity measures. To ensure that no objects obstruct the tag detection and to achieve the most stable detection possible, it is highly recommended to also prepare fixed mounts for these tags, as shown below.
+
+Note that the ones we used are lidartags of size 0.8 meters. This means the apriltags payload is 0.6 meters. We have also tried with lidartags of sizes 0.6 meters, but to use them the user needs to set several parameters by himself.
+
+<p align="center">
+    <img src="../docs/images/tag_based_pnp_calibrator/lidartag-mount.jpg"  alt="lidartag-mount" width="500">
+</p>
+
+## References
+
+[1] Jiunn-Kai (Bruce) Huang, Shoutian Wang, Maani Ghaffari, and Jessy W. Grizzle, "LiDARTag: A Real-Time Fiducial Tag System for Point Clouds," in IEEE Robotics and Automation Letters. Volume: 6, Issue: 3, July 2021.
+
+[2] G. Terzakis and M. Lourakis, "A Consistently Fast and Globally Optimal Solution to the Perspective-n-Point Problem" in ECCV 2020. Lecture Notes in Computer Science, vol 12346.
+
+## Known issues/limitations
+
+- The tool uses a basic OpenCV camera model for calibration (plumb bomb).
```

Comment:
```suggestion
- The tool uses a basic OpenCV camera model for calibration (plumb bomb).
- Only lidars with an intensity channel can be used (lidartag requirement)
- Only traditional lidars with a rotational scan pattern are compatible (other lidars may be compatible but we provide no guarantees)
```

Reply:
added in https://github.com/tier4/CalibrationTools/pull/164/commits/d9b2da19110b94625ccf2d58675ff2ef33e085c3

Reply:
Did you read correctly what you commited?

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/2c6ec5a56e700b27e5f856287aacea3894ba2d12